src/settings.py

```diff
@@ -87,9 +87,17 @@ class GlobalSettings:
     behavior_scene_name = "Pomaria_1_int"
     behavior_randomize_init_state = False
 
-    # parameters for approaches
+    # parameters for random options approach
     random_options_max_tries = 100
 
+    # parameters for GNN policy approach
+    gnn_policy_num_message_passing = 3
+    gnn_policy_learning_rate = 1e-3
+    gnn_policy_num_epochs = 2500
```

Comment:
@tomsilver this parameter is very important. it used to be 500 in other codebases, but increasing it to 2500 helped a LOT

Reply:
good to know